src/ReflectionFileNamespace.php

```diff
@@ -357,6 +362,20 @@ private function findConstants()
             }
         }
 
+        if ($withDefinedConstants) {
+            foreach ($this->namespaceNode->stmts as $namespaceLevelNode) {
+                if ($namespaceLevelNode instanceof FuncCall && (string)$namespaceLevelNode->name === 'define') {
```

Comment:
You can not cast name to the string, because according to the BNF, name can be instance of `PhpParser\Node\Expr` and it doesn't contain `__toString()` method. So, it will be better to check that our argument can be cast to the string and then collect it's name.

I can suggest to implement this logic in the node visitor for collecting defines. See classes in the `Go\ParserReflection\NodeVisitor` namespace for examples.